scripts/release/homebrew/docker/run.sh

```diff
@@ -7,8 +7,6 @@ pip install -U pip
 pip install -r $root/requirements.txt
 pip install azure-cli==$CLI_VERSION -f /mnt/pypi
 
-pip install msrestazure==0.6.2
-
```

Comment:
why remove it?

Reply:
1. We are using 0.6.3 now.
2. This line is redundant. We have defined dependencies.